app/models/miq_report_result.rb

```diff
@@ -26,10 +26,10 @@ class MiqReportResult < ActiveRecord::Base
 
   include ReportableMixin
 
-  delegate :table, :to => :report_results
+  delegate :table, :to => :report_results, :allow_nil => true
 
   def result_set
-    table.map(&:to_hash)
+    (table || []).map(&:to_hash)
```

Comment:
unfortunately this doesn't work since `table` here is not an array - it calls `to_ary` on it which raises an exception: 

```
Ruport::Controller::UnknownFormatError: Ruport::Controller::UnknownFormatError
from /home/tim/src/manageiq/vendor/bundle/ruby/2.2.0/bundler/gems/ruport-3ba69305f42e/lib/ruport/controller.rb:171:in `as'

```